[0.000000] (-) TimerEvent: {}
[0.000239] (NIDS_PKG) JobQueued: {'identifier': 'NIDS_PKG', 'dependencies': OrderedDict()}
[0.000298] (NIDS_PKG) JobStarted: {'identifier': 'NIDS_PKG'}
[0.098675] (-) TimerEvent: {}
[0.199040] (-) TimerEvent: {}
[0.299353] (-) TimerEvent: {}
[0.399713] (-) TimerEvent: {}
[0.499951] (-) TimerEvent: {}
[0.600151] (-) TimerEvent: {}
[0.629136] (NIDS_PKG) StderrLine: {'line': b'Traceback (most recent call last):\n  File "/usr/lib/python3/dist-packages/colcon_parallel_executor/executor/parallel.py", line 67, in execute\n    loop.run_until_complete(future)\n  File "/usr/lib/python3.8/asyncio/base_events.py", line 603, in run_until_complete\n    self.run_forever()\n  File "/usr/lib/python3.8/asyncio/base_events.py", line 570, in run_forever\n    self._run_once()\n  File "/usr/lib/python3.8/asyncio/base_events.py", line 1823, in _run_once\n    event_list = self._selector.select(timeout)\n  File "/usr/lib/python3.8/selectors.py", line 468, in select\n    fd_event_list = self._selector.poll(timeout, max_ev)\nKeyboardInterrupt\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/usr/lib/python3/dist-packages/colcon_core/executor/__init__.py", line 91, in __call__\n    rc = await self.task(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/__init__.py", line 93, in __call__\n    return await task_method(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_ros/task/ament_python/build.py", line 105, in build\n    return await extension.build(additional_hooks=additional_hooks)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/build.py", line 81, in build\n    available_commands = await self._get_available_commands(\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/build.py", line 171, in _get_available_commands\n    output = await check_output(\n  File "/usr/lib/python3/dist-packages/colcon_core/subprocess.py", line 128, in check_output\n    assert not rc, f\'Expected {args} to pass: {stderr_data}\'\nAssertionError: Expected [\'/usr/bin/python3\', \'setup.py\', \'--help-commands\'] to pass: Traceback (most recent call last):\n  File "setup.py", line 5, in <module>\n    setup(\n  File "/usr/lib/python3/dist-packages/setuptools/__init__.py", line 144, in setup\n    return distutils.core.setup(**attrs)\n  File "/usr/lib/python3.8/distutils/core.py", line 134, in setup\n    ok = dist.parse_command_line()\n  File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 707, in parse_command_line\n    result = _Distribution.parse_command_line(self)\n  File "/usr/lib/python3.8/distutils/dist.py", line 480, in parse_command_line\n    if self.handle_display_options(option_order):\n  File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 1092, in handle_display_options\n    return _Distribution.handle_display_options(self, option_order)\n  File "/usr/lib/python3.8/distutils/dist.py", line 683, in handle_display_options\n    self.print_commands()\n  File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 845, in print_commands\n    return _Distribution.print_commands(self)\n  File "/usr/lib/python3.8/distutils/dist.py", line 752, in print_commands\n    self.print_command_list(std_commands,\n  File "/usr/lib/python3.8/distutils/dist.py", line 720, in print_command_list\n    klass = self.get_command_class(cmd)\n  File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 837, in get_command_class\n    return _Distribution.get_command_class(self, command)\n  File "/usr/lib/python3.8/distutils/dist.py", line 828, in get_command_class\n    __import__(module_name)\n  File "/usr/lib/python3.8/distutils/command/check.py", line 10, in <module>\n    from docutils.utils import Reporter\n  File "<frozen importlib._bootstrap>", line 991, in _find_and_load\n  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked\n  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked\n  File "<frozen importlib._bootstrap_external>", line 844, in exec_module\n  File "<frozen importlib._bootstrap_external>", line 948, in get_code\n  File "<frozen importlib._bootstrap_external>", line 574, in _classify_pyc\nKeyboardInterrupt\n\n'}
[0.629750] (NIDS_PKG) JobEnded: {'identifier': 'NIDS_PKG', 'rc': 'SIGINT'}
[0.639557] (-) EventReactorShutdown: {}
